addons/mail/static/src/models/channel_member_view.js

```diff
@@ -27,6 +28,13 @@ registerModel({
             }
             this.channelMember.partner.openProfile();
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computePartnerImStatusIconView() {
+            return this.channelMember.partner && this.channelMember.partner.im_status && this.channelMember.partner.im_status !== 'im_partner' ? insertAndReplace() : clear();
```

Comment:
Could introduce `isImStatusSet` field, to avoid to duplicate this expression several times

Reply:
ok